docs/installation/config/openorganisatie_config.rst

```diff
@@ -0,0 +1,124 @@
+.. _installation_configuration:
+
+======================================
+Open Organisatie configuration (admin)
+======================================
+
+Before you can work with Open Organisatie after installation, a few settings need to be
+configured first.
+
+.. note::
+
+    This document describes the manual configuration via the admin.
+
+.. _installation_configuration_notificaties_api:
+
+Configure Notificaties API
```

Comment:
`SITE_DOMAIN` became required in notifications-api-common 0.9.0 and should probably be mentioned here.